vignettes/eval_vignette.Rmd

```diff
@@ -138,33 +137,75 @@ getR <- EpiEstim::estimate_R(
   backimputation_window = 10
 )
 
-#
-INCUBATION_SHIFT = round(weighted.mean(x = all_data$incubation$Day,
-                                       w = all_data$incubation$Px))
-
-REPORTINGDELAY_SHIFT = round(weighted.mean(x = all_data$reporting_delay$Day,
-                                           w = all_data$reporting_delay$Px))
-
-# PLOT DATA
-plot_data <- data.frame( ## ANNE
-  package = "EpiEstim",
-  date = all_data$cases$day[getR$R$t_end] - INCUBATION_SHIFT - REPORTINGDELAY_SHIFT,
-  Rt_median = getR$R$`Median(R)`,
-  Rt_lb = getR$R$`Quantile.0.025(R)`,
-  Rt_ub = getR$R$`Quantile.0.975(R)`
+```
+
+```{r rtestim}
+res_rtestim <- cv_estimate_rt(
+  observed_counts = all_data$cases$daily_reports[rr],
```

Comment:
Might want @dajmcdon  to review this to make sure it is being handled correctly in this vignette.